### PR TITLE
Update /v1/responses and /v1/declines and edit survey props

### PR DESCRIPTION
**Pull request by alfonsocora**
## Commits
- Convert `responses.md` & `declines.md` to unix format. Nobody is using Windows and we don't want those `^M` characters all over our line endings. This is split into a separate commit for easier review (the important stuff is in the next commit).
- Update docs for `/v1/responses` & `/v1/declines`
  * Add the `end_user.properties` fields
  * Fix the order of the fields
  * Update all the payloads
  * Add information on how to send properties when creating a response or decline
  * Add `PUT /v1/responses/:id` and `PUT /v1/declines/:id` including information on how to update survey properties

### CODE REVIEW

File: source/includes/_declines.md [hunk 1]
```diff
@@ -1,211 +1,277 @@
...
+
+This endpoint creates a decline for the end user.
+
+### HTTP Request
+
+`POST https://api.wootric.com/v1/end_users/<end_user_id>/declines`
+
+### URL Parameters
+
+Parameter | Description
```
Comment (diegoserranoa) [inline source/includes/_declines.md]: @alfonsocora maybe we should include `Type` also? Like on the other ones...

File: source/includes/_responses.md [hunk 2]
```diff
@@ -1,407 +1,570 @@
...
+score | integer | The response's score
+text | text | The response's comment
+completed | boolean | Specifies if a response has been completed
+excluded_from_calculations | boolean | Specifies if a response should be excluded from calculations
+ip_address | text | The response's IP address
+origin_url | text | The response's origin url
+created_at | datetime | Datetime representation of when the response was created
+updated_at | datetime | Datetime representation of when the response was last updated
+tags | string array | The responses's tags
+end_user.properties | object | The end user properties at the moment the survey was taken
```
Comment (diegoserranoa) [inline source/includes/_responses.md]: object or hash?

Comment (alfonsocora) [inline source/includes/_responses.md]: it looks like we're using hash everywhere, so I'll change it. Good catch!

**Comment by alfonsocora** [comment]
@diegoserranoa I addressed your comments. Would you mind taking another look? Thanks!

**Comment by diegoserranoa** [comment]
@alfonsocora looks good! :shipit:

**Pull request merged**